CHANGES.rst

```diff
@@ -10,6 +10,9 @@ New Features
 - Dropdown menus generated by ``plugin_select.vue`` now have the ability to be searched when
   enabled. [#3729]
 
+- Source catalogs - anything that can be can represented as an astropy table
+  and has RA and Dec or SkyCoord columns - be loaded into the data collection. [#3761]
+
```

Comment:
I wonder if we should keep this out of the changelog for now since it will be behind a dev flag, and maybe just mention in the follow-up tickets to include this PR on any future changelog entry?